viewing_party/main.py

```diff
@@ -0,0 +1,115 @@
+#WAVE 1 *******************************************************************************************************************************************************
+
+def create_movie(title, genre, rating):
+    while title and genre and rating:
```

Comment:
since there is no iteration happening here, I'd suggest using a conditional statement